core/payment/src/processor.rs

```diff
@@ -604,6 +606,10 @@ impl PaymentProcessor {
         Ok(result)
     }
 
+    pub async fn release_allocations(&self, db: Data<DbExecutor>) {
+        allocations::release_allocations(db, true).await
```

Comment:
looking a few lines above the database is accessed like this:
```
let existing_allocations = self
            .db_executor
            .as_dao::<AllocationDao>()
            .get_for_address(platform.clone(), address.clone())
            .await?;
```

would keep this structure to make all code consistent